classes/NPRAPIWordpress.php

```diff
@@ -94,10 +94,24 @@ function query_by_url( $url ) {
    * This function will go through the list of stories in the object and check to see if there are updates
    * available from the NPR API if the pubDate on the API is after the pubDate originally stored locally.
    *
+   * @see https://github.com/nprds/nprapi-wordpress/commit/ac0a7f7e3b7428ba783853eb76696b499bc85ecc for is_numeric($args)
+   *
+   * @since 1.7
+   *
    * @param bool $publish
+   * @param array|int $opts { If numeric it will be `'query_number' otherwise an array of named options.
+   *
+   *    @type int $query_number The number of the query for $args['tags_input']
+   *
+   * }
    * @return int|null $post_id or null
    */
-    function update_posts_from_stories( $publish = TRUE ) {
```

Comment:
If you could, resubmit your PR with the most recent `master` branch, which has the `$qnum` parameter added to this function instead of an `$opts` array or WP arg string